tests/publiccloud/ipa.pm

```diff
@@ -55,6 +77,24 @@ sub run {
         my $startup_timings = extract_startup_timings($out);
         record_info('Kernel boot is too slow',         result => 'fail') if $startup_timings->{'kernel'} > $kernel_max_boot_time;
         record_info('Overall system boot is too slow', result => 'fail') if $startup_timings->{'overall'} > $system_max_boot_time;
+        my $url = get_var('PUBLIC_CLOUD_PERF_DB_URI', 'http://openqa-perf.qa.suse.de:8086');
+        if ($url =~ m/^http/) {
```

Comment:
and if it is not containing http then what ? or remove the check or add some reaction in case it does not fit

Reply:
The reaction is, that it doesn't store the values in DB.
I think I'm going to remove the default URL. And check if PUBLIC_CLOUD_PREF_DB_URI is given and then call the `curl`.

Reply:
yeah this would looks better